### ISSUE TITLE
import error in test_provenance (and 3 others)

### ISSUE THREAD

**Issue by djarecka** [nipy/nipype#1798]
### Summary
Travis fails when running test_provenance (and 3 others), all pythons.

### Actual behavior
>   from lxml import etree
E   ImportError: libicui18n.so.56: cannot open shared object file: No such file or directory

see travis output for more details: 
https://travis-ci.org/djarecka/nipype/jobs/198015225

### How to replicate the behavior
don't know how to replicate outside travis

**Comment by mgxd** [comment]
ran into this error in #1794 as well, adding icu directly to Travis's conda seemed to fix it but I don't know if this is the best fix

**Comment by djarecka** [comment]
@mgxd  - I also found that people add `icu` to conda, I don't have any other idea.

**Comment by djarecka** [comment]
@mgxd : so won't create any new PR, if yours includes it.

**Comment by mgxd** [comment]
I believe this is fixed with #1794, but we should monitor this

**Comment by mgxd** [comment]
this is still present despite adding icu

**Comment by mgxd** [comment]
closing this for now after #1808
